music21/midi/translate.py

```diff
@@ -515,6 +517,15 @@ def midiEventsToChord(eventList, ticksPerQuarter=None, inputM21=None):
     pitches = []
     volumes = []
 
+    def handleEvent(event, pitches, volumes, thisChord):
+        p = pitch.Pitch()
```

Comment:
Now that this is factored out, can it have docs and typings?  What are pitches?  pitch objects? etc.  I know it was just a way of reducing code duplication -- which is great! -- but not that it's moved out of context, it's harder to understand what is happening than before.  With docs and doctests and typing it's a big positive.